Makefile

```diff
@@ -6,16 +6,14 @@ GO_INSTALL = ./scripts/go_install.sh
 KUBE_VERSION?="v1.25.8"
 CLUSTER_NAME?="eks-operator-e2e"
 GIT_COMMIT?=$(shell git rev-parse HEAD)
-GIT_COMMIT_SHORT?=$(shell git rev-parse --short HEAD)
-GIT_TAG?=$(shell git describe --abbrev=0 --tags 2>/dev/null || echo "v0.0.0" )
-TAG?=${GIT_TAG}-${GIT_COMMIT_SHORT}
 REPO?=ghcr.io/rancher/eks-operator
 E2E_CONF_FILE ?= $(ROOT_DIR)/test/e2e/config/config.yaml
-CHART_VERSION?=$(subst v,,$(GIT_TAG))
-RAWCOMMITDATE=$(shell git log -n1 --format="%at")
+TAG?=9000.0.0-dev # Only used in e2e tests
```

Comment:
Trying to understand this a bit more: what is the motivation behind using the hardcoded tag if we can use commit ID of the current HEAD?